client/client.go

```diff
@@ -172,11 +172,7 @@ func (r *NotaryRepository) Initialize(rootKeyID string) error {
 	err = r.tufRepo.InitRoot(false)
 	if err != nil {
 		logrus.Debug("Error on InitRoot: ", err.Error())
-		switch err.(type) {
-		case signed.ErrInsufficientSignatures, trustmanager.ErrPasswordInvalid:
-		default:
-			return err
-		}
+		return err
```

Comment:
I am not sure what the original intent was either; but with the current code the patch can not change behavior because the only kind of error `InitRoot` can return is `data.ErrInvalidRole`.

(It is the `CryptoService.GetPrivateKey` called at the very start of `NotaryRepository.Initialize` which fails if the user can’t provide the correct passphrase; for that kind of failure the code doesn’t get this far anyway.)